app/model/entity/ReferenceSolutionSubmission.php

```diff
@@ -80,6 +48,13 @@ public function setEvaluation(SolutionEvaluation $evaluation) {
     $this->evaluation = $evaluation;
   }
 
+  /**
+   * @var Collection
+   * @ORM\OneToMany(targetEntity="SubmissionFailure", mappedBy="referenceSolutionSubmission")
+   */
+  protected $failures;
+
+
```

Comment:
Remove this empty line, please :D

Reply:
![No](https://vignette4.wikia.nocookie.net/cardfight/images/f/f7/NO..jpg/revision/latest?cb=20140128085000)